core/services/job/orm.go

```diff
@@ -260,6 +260,7 @@ func (o *orm) CreateJob(jb *Job, qopts ...pg.QOpt) error {
 				if err2 := o.assertBridgesExist(*feePipeline); err2 != nil {
 					return err2
 				}
+			case DKG, OCR2VRF:
```

Comment:
Ah interesting, looks like we forgot to add validation code here for these two job types. Good find!